setup.py

```diff
@@ -202,6 +202,11 @@ def run(self):
     :undoc-members:
 """ % mod)
 
+# (function, predicate), see http://docs.python.org/2/distutils/apiref.html#distutils.cmd.Command.sub_commands
```

Comment:
because this commit was identified as causing problems for the debian package: i simply reverted it.